### DISCUSSION TITLE
Alt鍵

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#363] [General]
在操作各種電腦軟體時，常會使用Alt鍵。下面的圖片，是我電腦鍵盤的Alt鍵。
<img width="858" alt="這個按鍵就是Alt鍵" src="https://user-images.githubusercontent.com/86505454/151920351-b2eb8b3e-308e-4d4a-b843-e8331193a9bf.png">
# Alt鍵的應用
<img width="858" alt="Alt鍵應用" src="https://user-images.githubusercontent.com/86505454/151920941-22c502fb-b518-43ac-9fa6-19e0d0a972a0.png">

**Comment by chupochun** [comment]
謝謝分享電腦按鍵的使用方法

**Comment by pekoranoinu** [comment]
原來alt鍵還有這種用法，真是長知識了。
